modules/network-observability-auth-multi-tenancy.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-auth-mutli-tenancy_{context}"]
+= Configure authorization for multi-tenancy
+Define `ClusterRole` and `ClusterRoleBinding`. The `netobserv-reader` in the `ClusterRole` can be used to enable multi-tenancy and allows individual user access, or group access, to the flows stored in Loki. You can create a YAML file to define these roles.  
```

Comment:
The `netobserv-reader` `ClusterRole` can be used.....
Dont think in the is needed

Reply:
Think parameter word is not needed